models/pathology_nuclei_segmentation_classification/configs/train.json

```diff
@@ -0,0 +1,502 @@
+            {
+                "_target_": "ComputeHoVerMapsd",
+                "keys": "label_inst"
+            },
+            {
+                "_target_": "Lambdad",
+                "keys": "label_inst",
+                "func": "$lambda x: x > 0",
+                "overwrite": "label"
+            },
+            {
+                "_target_": "CenterSpatialCropd",
+                "keys": [
+                    "label",
+                    "hover_label_inst",
+                    "label_inst",
+                    "label_type"
+                ],
+                "roi_size": [
+                    80,
+                    80
+                ]
+            },
+            {
+                "_target_": "AsDiscreted",
+                "keys": "label",
+                "to_onehot": 2
+            },
+            {
+                "_target_": "CastToTyped",
+                "keys": [
+                    "image",
+                    "label_inst",
+                    "label_type"
+                ],
+                "dtype": "$torch.float32"
+            }
+        ],
+        "preprocessing": {
+            "_target_": "Compose",
+            "transforms": "$@train#preprocessing_transforms"
+        },
+        "dataset": {
+            "_target_": "Dataset",
+            "data": "$[{'image': i, 'label_inst': j, 'label_type': k} for i, j, k in zip(@train_images, @train_inst_map, @train_type_map)]",
+            "transform": "@train#preprocessing"
+        },
+        "dataloader": {
+            "_target_": "DataLoader",
+            "dataset": "@train#dataset",
+            "batch_size": 16,
+            "shuffle": true,
+            "num_workers": 4
+        },
+        "inferer": {
+            "_target_": "SimpleInferer"
+        },
+        "postprocessing_np": {
+            "_target_": "Compose",
+            "transforms": [
+                {
+                    "_target_": "Activationsd",
+                    "keys": "nucleus_prediction",
+                    "softmax": true
+                },
+                {
+                    "_target_": "AsDiscreted",
+                    "keys": "nucleus_prediction",
+                    "argmax": true
+                }
+            ]
+        },
+        "postprocessing": {
+            "_target_": "Lambdad",
+            "keys": "pred",
+            "func": "$@train#postprocessing_np"
+        },
+        "handlers": [
+            {
+                "_target_": "LrScheduleHandler",
+                "lr_scheduler": "@lr_scheduler",
+                "print_lr": true
+            },
+            {
+                "_target_": "ValidationHandler",
+                "validator": "@validate#evaluator",
+                "epoch_level": true,
+                "interval": "@val_interval"
+            },
+            {
+                "_target_": "CheckpointSaver",
+                "save_dir": "@ckpt_dir",
+                "save_dict": {
+                    "net": "@network",
```

Comment:
you are producing `net` inferer is using `model`.. both needs to be in sync at the end of the day.. and let them follow again similar names/standards like others while saving the checkpoint

Reply:
@KumoLiu, I agree with @SachidanandAlle that we should be consistent with other bundles. But how long re-training would take?

Reply:
Actually, we can just rename the key in the checkpoint, and run a training later to double check it.